script.js

```diff
@@ -1,4 +1,46 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computerchoice_nb = randomObject();
+  var computerchoice = "";
+  if (computerchoice_nb == 1) {
+    computerchoice = "scissors";
+  }
+  if (computerchoice_nb == 2) {
+    computerchoice = "paper";
+  }
+  if (computerchoice_nb == 3) {
+    computerchoice = "stone";
+  }
+  var myOutputValue =
+    "The computer chose " +
+    computerchoice +
+    " <br> You chose " +
+    input +
+    ".<br><br>" +
+    "You lose! Bummer. <br><br>Now you can type 'scissors' 'paper' or 'stone' to play another round!";
+
+  if (
+    (input == "scissors" && computerchoice == "paper") ||
+    (input == "paper" && computerchoice == "stone") ||
+    (input == "stone" && computerchoice == "scissors") ||
+    (input == "reversed scissors" && computerchoice == "stone") ||
+    (input == "reversed paper" && computerchoice == "scissors") ||
+    (input == "reversed stone" && computerchoice == "paper")
+  ) {
+    var myOutputValue =
+      "The computer chose " +
+      computerchoice +
+      " <br> You chose " +
+      input +
+      ".<br><br>" +
+      "You Win! <br><br> Now you can type 'scissors' 'paper' or 'stone' to play another round!";
```

Comment:
The `myOutputValue` variable is already created using the `var` keyword in [lines 13 to 19](https://github.com/ngzongyi/basics-scissors-paper-stone/blob/6a5453c59e044994856ccfd145084ffb41ad0c75/script.js#L13-L19). It is not inside an `if` block in the `main` function, which means it will definitely be created when the button is clicked.

Thus, in [lines 29 to 35](https://github.com/ngzongyi/basics-scissors-paper-stone/blob/6a5453c59e044994856ccfd145084ffb41ad0c75/script.js#L29-L35), there is no need to re-create the `myOutputValue` variable again using the `var` keyword.

Instead, you need only re-assign the existing `myOutputValue` to the new string which states the player has won. Re-assignment means you are not re-creating `myOutputValue`, which also means you do not need the `var` keyword here. Consider the following example, without the `var` keyword:

```
if (
    (input == "scissors" && computerchoice == "paper") ||
    (input == "paper" && computerchoice == "stone") ||
    (input == "stone" && computerchoice == "scissors") ||
    (input == "reversed scissors" && computerchoice == "stone") ||
    (input == "reversed paper" && computerchoice == "scissors") ||
    (input == "reversed stone" && computerchoice == "paper")
  ) {
    myOutputValue =
      "The computer chose " +
      computerchoice +
      " <br> You chose " +
      input +
      ".<br><br>" +
      "You Win! <br><br> Now you can type 'scissors' 'paper' or 'stone' to play another round!";
  }
```